### PR TITLE
Bump hammer-cli-katello to 0.14.1

### PR DESCRIPTION
**Pull request by akofink**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -78,6 +78,9 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/test
 
 %changelog
+* Wed Aug 29 2018 Andrew Kofink <akofink@redhat.com> 0.14.0-1
```
Comment (akofink) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: @ekohl why doesn't jenkins like my changelog entry?

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 Summary: Katello command plugin for the Hammer CLI
 Name:    %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.13.4
+Version: 0.14.0
```
Comment (ehelms) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: @akofink You'll need to set the release back to 2 on line 8

**Comment by akofink** [comment]
Updated release to 1

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 3]
```diff
@@ -26,7 +26,7 @@ Obsoletes: rubygem-hammer_cli_gutterball
 
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.13.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.14.0
```
Comment (ekohl) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: I see the gemspec fully dropped the minimum version requirement. Not sure that's wise and I'd prefer to keep it in the gemspec.

**Pull request merged**